src/SessionManager.ts

```diff
@@ -198,7 +200,14 @@ export class SessionManager {
 
               resolve(this.waitForSession(session, etag, accessToken))
             } else {
-              reject('Could not get session state.')
+              reject(
```

Comment:
Shouldn't we have tests that assert that this happens?

Reply:
good catch, done